firefox_ui_tests/functional/locationbar/test_escape_autocomplete.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestEscapeAutocomplete(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Clear complete history so there's no interference from previous entries.
+        self.places.remove_all_history()
+
+        self.test_urls = [
+            'layout/mozilla.html',
+            'layout/mozilla_community.html',
+        ]
+        self.test_urls = [self.marionette.absolute_url(t)
+                          for t in self.test_urls]
+
+        self.test_string = 'mozilla'
+
+        self.locationbar = self.browser.navbar.locationbar
+        self.autocomplete_results = self.locationbar.autocomplete_results
+
+    def tearDown(self):
+        # workaround for marionette bug leaving us in 'content' context here
+        self.marionette.set_context('chrome')
```

Comment:
Chris suggested this workaround, noting there's a Marionette bug that leaves us in content context in tearDown.

Since the skip decorator works well applied to the class, that's the better solution here.